manifests/puppetca.pp

```diff
@@ -1,12 +1,4 @@
 # Configure puppet CA component
 class foreman_proxy::puppetca {
```

Comment:
this is connected in some ways in the mode by using `enabled   => $::foreman_proxy::puppetca,` and `<% if scope.lookupvar('foreman_proxy::puppetca')...`, so I'd like to keep this.